Add blog bost for April standard community call (#422)
publiccodenet/blog: _posts/2023-04-11-notes-from-community-call-6-april-2023.md

The strong point about what is in the control of a codebase was made again. This strongly suggests that the requirements related to search engines fit better as SHOULD or OPTIONAL rather than MUST requirements. The point applies to both requirements about search engines.

Related to what is mentioned in the GitHub issue, it was discussed if we should change from "describing the problem it solves in natural language." to something like "the marketing phrase the codebase describe itself with". But instead of changing the existing requirement there was an idea of adding a new requirement around a one line sentence describing the codebase that could be used wherever needed. This aligns well with the [community assets](https://about.publiccode.net/activities/codebase-stewardship/community-assets.html) we already have which calls for a "one sentence codebase description for repository platform". It was agreed that it would make sense to use that the existence of such a description is important enough to use that as the base for a new requirement.

> Finally, the other criteria about documentation were also discussed as what this documentation can be highly relevant to search engines. And as they may rank high in the search engines, it is important that they are packaged with, or prominently link to, the main place for the codebase.

Reviewer: ```suggestion
Finally, the other criteria about documentation were also discussed, as codebaase documentation can be highly relevant to search engines. As they may rank highly in search engines, it is important that they are packaged with, or prominently link to, the main home for the codebase.
```
